src/app/common/modals/calendar-modal/calendar-modal.component.scss

```diff
@@ -0,0 +1,31 @@
+.calendar-modal-component {
+  max-width: 600px;
+  overflow-x: hidden;
+
+  .d-flex {
+    display: flex;
+
+    &.d-flex-row {
+      flex-direction: row;
+    }
+
+    .d-flex-grow {
+      flex-grow: 1;
+    }
+  }
+
+  .w-100 {
+    width: 100%;
+  }
+
+  .webcal-url-container {
+    display: flex;
+    align-items: center;
+  }
+  
+  .webcal-options-card {
```

Comment:
This looks like a pretty generic card style, let's think about generalising this...